jenkins_job_wrecker/cli.py

```diff
@@ -26,22 +27,24 @@
 log = logging.getLogger('jjwrecker')
 
 
-def str_presenter(dumper, data):
-    if len(data.splitlines()) > 1:  # check for multiline string
+def get_str_presenter(should_replace_tabs=False):
+    def str_presenter(dumper, data):
         # The dumper will not respect "style='|'" if it detects trailing
         # whitespace on any line within the data. For scripts the trailing
         # whitespace is not important.
         lines = [l.rstrip() for l in data.splitlines()]
+        if should_replace_tabs:
+            # Dumper also will not work on multiline if there are tab escape
+            # characters. If multiline is desired on text that has tab escape
+            # character, tabs should be replaced with spaces.
+            lines = [replace_tab(l) for l in lines]
         data = '\n'.join(lines)
```

Comment:
should the whole section above be put under the if condition below, as it was in the past?

Reply:
I do believe that rsplit should not be applied for single line. I have made necessary changes.